R/simulate.R

```diff
@@ -477,3 +477,107 @@ simulate_chain_stats <- function(n_chains,
 
   return(out)
 }
+
+#' Calculate transmission chain statistics across a range of scenarios and
+#' divide into intervals
```

Comment:
I've left a comment below on the fact that the part about dividing into intervals is just one use case and so this function should probably return the raw results.